### ISSUE TITLE
how to do autoscroll in ng onboarding ?

### ISSUE THREAD

**Issue by sibascodeking** [adamalbrecht/ngOnboarding#15]

**Comment by novarac23** [comment]
@sibascodeking How do you mean auto scroll ? For example if you put a lot of text and pics and have the scrolling sidebar on the right ?

**Comment by sibascodeking** [comment]
No, while it moves from step 1 to step 2 it needs to be scroll and go down for the next step smoothly.
